### DISCUSSION TITLE
Autoscroll

### DISCUSSION THREAD

**Discussion by denis-demyanov** [sevdokimov/log-viewer#13] [Ideas]
День добрый!

Лог сам обновляется, но нет (не нашел?) возможности автопрокрутки к последней строке лога. Аналога "tail -F /var/log/my.log"
Вероятно выбор отображения "последняя_строка_свеху" / "последняя_строка_снизу" в принципе решила бы задачу.

Заранее благодарен.

**Comment by sevdokimov** [comment]
Добрый день,

Да, надо бы сделать автопрокрутку когда лог растёт.

**Reply by wnleao** [reply]
How do I turn on/off the autoscroll? I've just started using the logviewer and even though the log is growing the page is stuck where I've left it. Thank you!

**Reply by sevdokimov** [reply]
@kiniggit , unfortunately, it is not implemented yet.

**Reply by wnleao** [reply]
@sevdokimov no problem! thank you!
